pages/linux/vnstat.md

```diff
@@ -0,0 +1,23 @@
+# vnstat
+
+> A console-based network traffic monitor.
+
+- Display traffic summary for all interfaces:
+
+`vnstat`
+
+- Display traffic summary for eth0:
+
+`vnstat -i eth0`
```

Comment:
We use our special `{{token_syntax}}` to indicate values that the user can change :-)
```suggestion
`vnstat -i {{eth0}}`
```